Modules/Core/ImageFunction/test/itkGaussianInterpolateImageFunctionTest.cxx

```diff
@@ -65,13 +65,13 @@ int itkGaussianInterpolateImageFunctionTest( int, char*[] )
   image->SetOrigin(origin);
   image->SetSpacing(spacing);
 
-  for ( unsigned int i = 0; i < 3; ++i )
-  {
-    for ( unsigned int j = 0; j < 3; ++j )
+  for ( itk::IndexValueType i = 0; i < 3; ++i )
     {
+    for ( itk::IndexValueType j = 0; j < 3; ++j )
+      {
       image->SetPixel({{i, j}}, i + j);
+      }
     }
```

Comment:
Looks like you could do it with a single range-based for-loop, instead of two old-style for-loops:

      using itk::Experimental::ZeroBasedIndexRange;

      for (const auto index : ZeroBasedIndexRange<ImageType::ImageDimension>(size))
      {
        image->SetPixel(index, index[0] + index[1]);
      }

But well... I approved your fix already  `:-)`

Reply:
Beeeeeautiful, @N-Dekker -- adding! :-)